.changeset/wise-goats-trade.md

```diff
@@ -0,0 +1,7 @@
+---
+'myst-to-react': patch
+'@myst-theme/site': patch
+'@myst-theme/book': patch
+---
+
+Fix margins on admonitions and site footer
```

Comment:
@tavin if you want in the future run:

`npm run changeset`

And it will add a few things to keep track of what has changed.